homework/index.js

```diff
@@ -29,23 +31,115 @@
     return elem;
   }
 
-  function renderRepoDetails(repo, ul) {
-    createAndAppend('li', ul, { text: repo.name });
+  function renderRepoTable(table, name, path) {
+    const tr = createAndAppend('tr', table);
+    createAndAppend('th', tr, {
+      text: name,
```

Comment:
The first column of the table containing the label text looks slightly better if you align them to the left. You can achieve that by adding another attribute for the `th` element: `align: 'left',`

Reply:
I added it to the CSS part, since I have not written any styling in .js page. Hope it is also ok like this.